index.d.ts

```diff
@@ -3,13 +3,16 @@
 
 /* auto-generated by NAPI-RS */
 
+export interface Pid {
+  pid: number
+}
 export interface Size {
   cols: number
   rows: number
 }
 export class Pty {
```

Comment:
does napi-rs preserve comments? it would be neat to have a little warning in the docstring telling folks to always start reading from the FD before calling `start` to avoid surprises.

Reply:
hm, this file is auto-generated, not sure if we can inject comments into it